src/content/content_manager.cc

```diff
@@ -238,15 +238,15 @@ void ContentManager::run()
             inotify->monitor(adir);
             auto param = std::make_shared<Timer::Parameter>(Timer::Parameter::timer_param_t::IDAutoscan, adir->getScanID());
             log_debug("Adding one-shot inotify scan");
-            timer->addTimerSubscriber(this, 60, param, true);
+            timer->addTimerSubscriber(this, std::chrono::minutes(1), param, true);
         }
     }
 #endif
 
     for (size_t i = 0; i < autoscan_timed->size(); i++) {
         std::shared_ptr<AutoscanDirectory> adir = autoscan_timed->get(i);
         auto param = std::make_shared<Timer::Parameter>(Timer::Parameter::timer_param_t::IDAutoscan, adir->getScanID());
-        log_debug("Adding timed scan with interval {}", adir->getInterval());
+        log_debug("Adding timed scan with interval {}", adir->getInterval().count());
```

Comment:
Format supports printing `chrono` stuff if you include the header

Reply:
Right. The ostream header also supports printing fs::path directly. I'm intentionally avoiding including it to avoid bloating the code.